### DISCUSSION TITLE
Deploying bot to Netlify/Vercel

### DISCUSSION THREAD

**Discussion by orangecoloured** [yagop/node-telegram-bot-api#1204] [Q&A]
I was creating a bot locally, so it was starting with `{ polling: true }`. Then when it was time to deploy I understood that it's not a usual deployment. It should be a serverless one. And atm I'm a bit lost. I see there are [examples](https://github.com/yagop/node-telegram-bot-api/tree/master/examples/webhook) for different platforms. But neither Vercel not Netlify are there. And these are the ones that I'm most familiar with.

Is there a guide using `node-telegram-bot-api` to deploy a bot with webhooks to one of these platforms?
But actually any serverless deployment guide using this package will do.

**Comment by Zoomish** [comment]
Hi! You should try this https://www.marclittlemore.com/serverless-telegram-chatbot-vercel, but I do not recommend that you post your hot on vercel/netlify, etc . You'd better use something like Render (Render.com ), he has a free tariff, which is more than enough in most cases.
